Jupyter Notebook Data Curation Primer/Jupyter Notebooks Data Curation Primer.md

```diff
@@ -100,10 +100,10 @@ The following elements outline recommendations for repositories accepting Jupyte
 - Additional files to request:
   - PDF of the Jupyter Notebook (export from Jupyter web application or [nbviewer](https://nbviewer.jupyter.org/))
   - reST export of the Jupyter Notebook (export from Jupyter web application)
-  - CodeMeta.json
-  - CITATION.cff
+  - CodeMeta.json, requirements.txt, or environment.yml (dependencies)
```

Comment:
I would recommend listing CodeMeta.json as preferred at least as it provides the ability to define more extensive structured metadata using a controlled vocab.